sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -1036,52 +1050,26 @@ private static String getUserAgent() {
   }
 
   /**
-   * A helper function to add appropriate bulk options. See
-   * <a href="https://github.com/GoogleCloudPlatform/cloud-bigtable-client/issues/899">RetryOptions
-   * toBuilder</a> for issue.
+   * A helper function to convert a RetryOptions into a RetryOptions.Builder.
```

Comment:
Add a TODO: remove this and use RetryOptions#toBuilder once updated Bigtable library is released.